backend/src/models/structure.js

```diff
@@ -0,0 +1,6 @@
+const mongoose = require('mongoose')
+
+const Structure = new mongoose.Schema({
```

Comment:
Maybe add a comment to describe what `Structure` is? Just because Structure is a little vague